components/servo/fake-ld.sh

```diff
@@ -9,7 +9,25 @@ set -o nounset
 set -o pipefail
 
 TARGET_DIR="${OUT_DIR}/../../.."
-arm-linux-androideabi-gcc "${@}" \
-                          "${LDFLAGS-}" -lc -shared \
-                          -o "${TARGET_DIR}/libservo.so"
-touch "${TARGET_DIR}/servo"
+
+export _ANDROID_ARCH=arch-arm
+export ANDROID_SYSROOT="$ANDROID_NDK/platforms/android-18/$_ANDROID_ARCH"
```

Comment:
Please use the full `"${ANDROID_NDK}/platforms ...` quoting style per the [style guide](https://github.com/servo/servo/blob/master/docs/STYLE_GUIDE.md)